auth-passwd.c

```diff
@@ -298,9 +298,24 @@ sys_auth_passwd(struct ssh *ssh, const char *password)
 	wchar_t * backslash = wcschr(user_utf16, L'\\');
 	if (backslash != NULL) {
 
+		/* assemble the path to the name translation library */
+		wchar_t library_path[MAX_PATH + 1];
```

Comment:
Understood.  I think my assumption (which may be incorrect), is that a OneCore based machine probably doesn't support being domain joined since there really doesn't seem to be domain-related APIs available.  Unless the machine is domain-joined, this code will never be hit.

Reply:
As far as the info I got, there is restrictions for onecore image to domain join.

Reply:
This is the answer I got from onecore sdk folks:
It is certainly true today that OneCore-based images cannot join an Active Directory domain.  (They *can* do Azure Active Directory operations, like the “Add a work account” flow on Windows Phone.)  It would be a lot of work to allow Active Directory join on OneCore.  I can’t promise that it will never be possible, but it is not possible today.

For that, I think it is fine keep the way loading TranslateNameW() dynamically and fall back to netbios domain name.